clang/lib/CIR/CodeGen/CIRGenFunction.cpp

```diff
@@ -306,6 +304,7 @@ LValue CIRGenFunction::emitLValue(const Expr *e) {
     getCIRGenModule().errorNYI(e->getSourceRange(),
                                std::string("l-value not implemented for '") +
                                    e->getStmtClassName() + "'");
+    return LValue();
     break;
```

Comment:
Since you added the `return`, I think you should get rid of the `break;`, which is now dead code.